core/dbt/config/project.py

```diff
@@ -135,7 +142,7 @@ def package_config_from_data(packages_data):
         packages = PackageConfig.from_dict(packages_data)
     except ValidationError as e:
         raise DbtProjectError(
-            MALFORMED_PACKAGE_ERROR
+            MALFORMED_PACKAGE_ERROR.format(error=str(e))
```

Comment:
Ooops - I think I led you astray here - we're actually going to want `str(e.message)`

```suggestion
            MALFORMED_PACKAGE_ERROR.format(error=str(e.msg))
```

In practice, this should result in output like:

![Screen Shot 2020-01-31 at 10 45 06 AM](https://user-images.githubusercontent.com/1541139/73552739-c7c88d00-4416-11ea-90f4-1ac2ba02dc3b.png)

Which i think is pretty good all things considered!

Note: we'll also want to remove the duplicated link on line 71!

Reply:
Thanks for the review!